pipe.go

```diff
@@ -18,54 +24,61 @@ type Pipe interface {
 	Commit(*Message) error
 }
 
-// ProcessorPipe represents the pipe for processors.
-type ProcessorPipe struct {
-	node  Node
-	queue []NodeMessage
+var _ = (TimedPipe)(&processorPipe{})
```

Comment:
Same check for base Pipe interface?

Reply:
Not needed as `NewPipe` returns the `Pipe` interface.